npm/src/updateconsignmentstatus/index.ts

```diff
@@ -12,8 +12,7 @@ export class UpdateConsignmentStatus {
   ): Promise<number | void | Error> {
     const updateConsignmentStatusInput: ConsignmentStatusInput = {
       consignmentId: uploadFilesInfo.consignmentId,
-      statusType: "Upload",
-      statusValue: "Completed"
+      statusType: "Upload"
```

Comment:
Can we rename the main method `markUploadStatusAsCompleted` to something more meaningful, maybe `setUploadStatusBasedOnFileStatuses`?